src/Multiselect/MultiselectComponents.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import classNames from 'classnames';
+import { components } from 'react-select';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+import { Close, KeyboardArrowDown } from '../../icons';
+
+const ClearIndicator = (props) => (
+  <components.ClearIndicator {...props}>
+    <Icon src={Close} />
```

Comment:
I'm curious if we should be using an `IconButton` here instead of a regular `Icon` to get its focus/hover styles, though I imagine it might mess with the spacing inside of the input. There might also be a11y implications for rendering a clickable icon outside of an element that indicates its an interactive element (i.e., its not currently inside of a `button` element or an element with a `role="button"` for example).

Reply:
Used `IconButton` instead of default component. Styled to fit the design. Added some changes according to focus/hover.